src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -363,57 +362,42 @@ public bool IsHomeSpace
         [JsonIgnore]
         internal JObject JsonRepresentation { get; set; }
 
+        [JsonIgnore]
+        internal string CurrentCheckSum { get; set; }
+
         /// <summary>
-        /// Returns the stringified representation of the connected nodes
+        /// Returns the stringified representation of the node connections in the workspace.
         /// </summary>
         [JsonIgnore]
         public string Checksum
         {
             get
             {
                 List<string> nodeInfoConnections = new List<string>();
-                JObject jsonWorkspace = JsonRepresentation;
```

Comment:
can you get rid of `JsonRepresentation`? - I think it may have been added just for this method...

Reply:
It is used here: https://github.com/DynamoDS/Dynamo/blob/bef8e58e09504660bd765fbe58161545f2da92b6/src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs#L755